### PR TITLE
Indent all Info and Warn log messages to align when printed.

### PR DESCRIPTION
**Pull request by mogren**
*Description of changes:*

Make [INFO] and [WARN] log lines align with [DEBUG] and [ERROR]

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/k8sapi/discovery.go [hunk 1]
```diff
@@ -276,7 +276,7 @@ func (c *controller) handleErr(err error, key interface{}) {
 
 	// This controller retries 5 times if something goes wrong. After that, it stops trying.
 	if c.queue.NumRequeues(key) < 5 {
-		log.Infof("Error syncing pod %v: %v", key, err)
+		log.Infof(" Failed syncing pod %v: %v", key, err)
```
Comment (tiffanyfay) [inline pkg/k8sapi/discovery.go]: Failed to*

File: plugins/routed-eni/driver/driver.go [hunk 2]
```diff
@@ -306,7 +306,7 @@ func tearDownNS(addr *net.IPNet, table int, netLink netlinkwrapper.NetLink) erro
 	if err != nil {
 		log.Errorf("Failed to delete toContainer rule for %s err %v", addr.String(), err)
 	} else {
-		log.Infof("Delete toContainer rule for %s ", addr.String())
+		log.Infof(" Delete toContainer rule for %s ", addr.String())
```
Comment (tiffanyfay) [inline plugins/routed-eni/driver/driver.go]: Deleted*

File: plugins/routed-eni/driver/driver.go [hunk 3]
```diff
@@ -316,7 +316,7 @@ func tearDownNS(addr *net.IPNet, table int, netLink netlinkwrapper.NetLink) erro
 			log.Errorf("Failed to delete fromContainer for %s %v", addr.String(), err)
 			return errors.Wrapf(err, "delete NS network: failed to delete fromContainer rule for %s", addr.String())
 		}
-		log.Infof("Delete fromContainer rule for %s in table %d", addr.String(), table)
+		log.Infof(" Delete fromContainer rule for %s in table %d", addr.String(), table)
```
Comment (tiffanyfay) [inline plugins/routed-eni/driver/driver.go]: Deleted*

**Comment by nckturner** [comment]
Instead of inspecting every added log line for number of spaces, can we add a tab to the logger format here? https://github.com/aws/amazon-vpc-cni-k8s/blob/master/pkg/utils/logger/logger.go#L37

**Comment by mogren** [comment]
Heh, that does sound like a much better idea. I'll take a look :)

**Pull request closed without merging**